### DISCUSSION TITLE
Moveable floating action button that remembers last set location

### DISCUSSION THREAD

**Discussion by xbxbruce** [Docile-Alligator/Infinity-For-Reddit#1599] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

First of all thanks to the dev for adding this function, the only problem is that if i move the button anywhere in the screen, the next post i see it will be resetted to the initial position. Would be awesome like the official reddit app and other apps.

### Alternatives

Not in the settings

### Additional context

I think this is needed as I'm a left handed person and every time it gets reset to the right side :( 

### How important is this feature?

Nice to have

**Comment by CynthiaTattersail** [comment]
Yeah I really wished it would remember the position. It's the one feature from the terrible official app I miss
